### ISSUE TITLE
Should licence continue to be a field that is require by default?

### ISSUE THREAD

**Issue by jcoyne** [projecthydra-labs/hyrax#764]
Now that rights is required, shall we relegate license to being optional?

<img width="795" alt="screen shot 2017-04-11 at 10 31 43 am" src="https://cloud.githubusercontent.com/assets/92044/24917222/18f9996e-1ea2-11e7-9c5f-b75ed8ce6f65.png">

**Comment by aaron-collier** [comment]
I vote that license be optional. Most of my institutions don't use any kind of CC license (etc) and just have a local rights statement.

**Comment by jlhardes** [comment]
We just had this conversation in the metadata committee for our DPLA Hub. Rights is more easily declared than licensing (which can require some legal work in determining). I think it makes sense to keep license separate and make it optional, if known. And requiring a rights statement helps in sharing to aggregators like DPLA, so that is great to require!

**Comment by cmh2166** [comment]
imho, keep rights as required, make license optional, perhaps we can update any relevant docs to recommend use of rightsstatements.org rights URIs (or labels if URIs aren't supported for this field) to clarify difference (vis a vis examples) between rights + licenses and the ability to use something like 'copyright not evaluated'.

**Comment by ruthtillman** [comment]
++ Julie and Christina, especially Christina's statement about supporting docs to help people understand that difference.

**Comment by chrissyrissmeyer** [comment]
I would personally vote for requiring a rightsstatement, but not a license. primarily because IMHO you could end up with some mis-use of licenses. for example, "In copyright - educational use permitted" doesn't really have an appropriate match on the license side

**Comment by mjgiarlo** [comment]
Tagging the new Hyrax metadataists group for advice! @projecthydra-labs/hyrax-metadataists

**Comment by chrissyrissmeyer** [comment]
This might be controversial, but another option would be to collapse into a single field (to be fully transparent this is what we currently in ADRL). If I remember correctly there was a lot of discussion around this in the rights metadata WG a couple of years ago. Creative Commons licenses are for use with copyrighted works which implies that the object in question is under copyright ("licenses that enable the free distribution of an otherwise copyrighted work"). See also ttps://wiki.duraspace.org/display/hydra/Rights+Metadata+Recommendation.

**Comment by cmh2166** [comment]
@chrissyrissmeyer what you propose seems to be focused more on expectations of the two fields and interactions (if not sure of one, use other only) type deal, as well as indicating a need for more docs somewhere in Hyrax (or other) wiki explaining how to apply each field. 

So I'm -1 to combining the two fields right now, bc I think they each have a role to play in the current minimal Hyrax metadata profile and suffer from lack of implementation guidance/documentation, not unclarity of purpose here. I'll propose the following to solve this issue but keep the surfaced discussions going somewhere else where we can make headway as needed or change or proposal + open a new issues later:

1. for @jcoyne to keep `rights` as required, but indeed make `license` optional;
2. for me to update the emerging Hyrax metadata-docs MAP to indicate this shift and link to the rights statement working group output;
3. for @chrissyrissmeyer or other to propose back to HMIG a longer form discussion there on improving metadata guidance documentation including the specific issue brought up above - don't apply a license if you're not sure, use rights statements to cover yourself otherwise, etc.

Is that okay?

**Comment by remerjohnson** [comment]
I am also -1 to combining `rights` and `license`, and +1 to keeping `license` optional. It's true that a CC license implies the work is in copyright, but I want to be explicit to the user what their options are. The text at the [rightsstatements.org](http://rightsstatements.org/en/) are (in my opinion) pretty clear in telling the user they need to adhere to any licenses or the permission of the copyright holder (and is also helpfully explicit that these statements are _not_ licenses).

**Comment by mark-dce** [comment]
**TL/DR**
I'm in agreement with the proposals to make `rights` required and `license` optional that are 

**DETAIL**
The use cases (Tufts, Case Western Reserve, WUSTL, UMich Publishing off the top of my head) I've seen often have highly unique and localized language for `license` which is often a very long free-form string.  Making `license` required seems like it would just encourage folks without specific practices to make up junk to fill in to this field - or as mentioned above, to conflate it with a rights statement.

Contrastingly, it seems reasonable and good long term management to ask someone to assert the high-level rights that pertain to a given item when it is being deposited rather than having to guess / fabricate / infer the rights at some future point.

**Comment by mjgiarlo** [comment]
I'm hearing consensus. Yay! Thanks, all.
